cas-server-integration-memcached/src/main/java/org/jasig/cas/ticket/registry/support/kryo/serial/ZonedDateTimeTranscoder.java

```diff
@@ -0,0 +1,29 @@
+package org.jasig.cas.ticket.registry.support.kryo.serial;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.Serializer;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import org.jasig.cas.util.DateTimeUtils;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+/**
+ * @author tduehr
+ * @since 4.3.0
+ */
+public class ZonedDateTimeTranscoder extends Serializer<ZonedDateTime> {
```

Comment:
The `JavaSerializer` class died horribly since there isn't a no arg constructor for `ZonedDateTime`. I looked around and got code similar to this almost immediately. I expect kryo will add something like this soon and it can be removed from CAS.